lnrpc/routerpc/router_server.go

```diff
@@ -0,0 +1,259 @@
+// +build routerpc
+
+package routerpc
```

Comment:
I just noticed these are all named "route-rpc", instead of "router-rpc" as I expected. Same for build flags. Is this intentional?

Reply:
It's a personal aesthetic thing for me: `routerpc` looks better than `routerrpc`. The consecutive "r" there makes it harder to parse.